framework/bin/run_mutation.pl

```diff
@@ -266,6 +272,17 @@ sub _run_mutation {
     $project->{prog_root} = "$root";
     $project->checkout_vid($vid) or die "Checkout failed";
 
+    my $num_excluded_mutants = 0;
+    if (defined $EXCL) {
+      system("cp $EXCL $root/") == 0 or die "Cannot copy exclude file";
+
+      # count number of mutants excluded
+      open(EXCL_FILE, "$EXCL");
+      my @str = <EXCL_FILE>;
+      close(EXCL_FILE);
+      $num_excluded_mutants = scalar(@str);
```

Comment:
The exclude file may contain comments or empty lines. Lines that match the following regexes are ignored:
`\s*#.*`
`\s*$`

I realize that this is poorly documented in Major.

Can you account for comments and empty lines when counting the number of excluded mutants?